src/server/controllers/item.controller.js

```diff
@@ -0,0 +1,37 @@
+const Item = require('../models/item.model');
+
+// Create new item
+const itemCreate = function(req, resp) {
+  const item = new Item({
+    name: req.body.name,
+    isOrdered: req.body.isOrdered
+  });
+
+  item.save(err => {
+    err
+      ? resp.status(400).send(err.message)
+      : resp.status(200).send('Product created successfuly!');
+  });
+};
+
+// Get item by given id
+const getItemById = function(req, resp) {
+  Item.findById(req.params.id, (err, product) => {
+    err ? resp.status(400).send(err.message) : resp.status(200).json(product);
+  });
+};
+
+// Delete item by given id
+const deleteItemById = function(req, resp) {
+  Item.find({ _id: req.params.id }).deleteOne((err, product) => {
```

Comment:
please check if findOneAndDelete or findByIdAndRemove could be used instead of calling two methods